public/gamecompare.php

```diff
@@ -196,7 +196,7 @@
                         $rightSoftcoreAwardedPoints += $achPoints;
                         echo "<small class='smalldate'>$awardedRight</small>";
                     }
-                    echo achievementAvatar($nextAch, label: false, icon: $badgeName . "_lock", iconSize: $iconSize, iconClass: 'awardremote', tooltip: false);
+                    echo achievementAvatar($nextAch, label: false, icon: $badgeName, iconSize: $iconSize, iconClass: isset($awardedHCRight) ? 'goldimage' : '', tooltip: false);
```

Comment:
`_lock` was incorrectly added by https://github.com/RetroAchievements/RAWeb/pull/1149/files#diff-1b3f2a3b9e72f81134ba1e00c5c39e9206122eb246ed2d1495c7ea923109d36bL199, but `awardremote` was there before. It doesn't seem to be defined, so I've replaced with the standard gold border for hardcore logic.

Reply:
Thanks! That was an oversight on my end